### DISCUSSION TITLE
Can't get started :-/

### DISCUSSION THREAD

**Discussion by bpolaszek** [ronengr/hass_nuki_bt#16] [Q&A]
Hello there,

I have a Nuki Smart Lock 1.0 without bridge and your repo looked like a fantastic promise!

Since my HA is a little far from the lock, I bought a bluetooth dongle to extend the range.
Then, when I got in Devices, the Nuki appeared. But when I clicked on Configure -> Pair automatically after setting the pair mode (by holding the main button until the light glows), I got an "An error occured" message.

I thought the HA was still a little far so I got it closer, but that message still appeared. So I just ignored the discovered Nuki and removed your repo from HACS.
Then I downloaded it again. 

But now, the Nuki never appears again, even if I put the HA box in the same room.

Any help would be much appreciated! 🙏

**Comment by nurimoca** [comment]
Hello, I have a similar issue. I managed to add the the nuki3, it appears on the integration but says its initializing and then shows an error...

**Reply by ronengr** [reply]
@nurimoca 
can you please add some logs?

**Reply by ronengr** [reply]
@bpolaszek 
sorry for the late reply.
I'm not sure how to force auto-discover to work again. This is an HA issue and not something specific to this integration.
maybe you clicked on ignore and now HA ignores this? can you check maybe in the disabled devices list?
another option is to add manually, if you know the BT address of the device.

**Comment by oliv3r** [comment]
I've just unboxed a Nuki Pro 5th Gen, and want to connect it to HA without having touched the app. Is this even possible at all?

I see it broadcasting 'Nuki_AABBCCDD' via bluetooth, so it's "there", but never got an auto-discover pop-up. Manually adding it via 'AABBCCDD" as device address didn't work (the window keeps popping to the 'name' field, like if it's not happy with the name, but won't tell me how/why/what is missing needed. The 'pin' is marked as optional, no idea what to put there either ...

**Reply by ronengr** [reply]
The HA integration was not build with such usage in mind.
There are many config options that are currently not supported by the integration and where supposed to be handled via the official app. This includes some initial setup of the device.
Once you initiated the lock and configured all the basic settings (setting lock type, how many rotations, what direction, etc) you can add it to HA using this integration.
